kit/cli/loglevelflag.go

```diff
@@ -0,0 +1,42 @@
+package cli
+
+import (
+	"fmt"
+	"github.com/spf13/pflag"
+
+	"go.uber.org/zap/zapcore"
+)
+
+type levelValue zapcore.Level
```

Comment:
Is this definition just a way to declare methods on the zapcore.Level type?  Nice trick!

Reply:
(I think) yes, specifically so we can redefine methods like `String` and `Set` that already exist on the zapcore.Level type.